### PR TITLE
Lite Refactoring

### PR DESCRIPTION
**Pull request by andela-joyebanji**

### CODE REVIEW

File: lib/superagent-mock.js [hunk 1]
```diff
@@ -142,43 +141,43 @@ function mock (superagent, config, logger) {
...
-          }
-        });
-        response.setStatusProperties(err.message);
-      }
-
-      fn(error, response);
-      return this;
-    } else {
-      oldEnd.call(this, fn);
+        res: {
```
Comment (DevSide) [inline lib/superagent-mock.js]: Could you bring the indentation back here ?

Comment (andela-joyebanji) [inline lib/superagent-mock.js]: @DevSide Nice catch. Done. 👍

**Comment by fdubost** [comment]
Thanks for your contribution ! 👍

**Comment by tzi** [comment]
Nice improvments!
Thanks :smile:

**Pull request merged**